orchestration/tests/test_cleanup_completed_agents.py

```
@@ -630,37 +609,33 @@ def test_cleanup_completed_agents_cleanup_failures_counted_correctly(
+    @patch("orchestration.cleanup_completed_agents.check_session_timeout")
+    def test_cleanup_completed_agents_only_task_agents_check_completion_logs(self, mock_timeout, mock_get_sessions):
         """Test only task-agent sessions check completion via logs."""
         # Arrange
         mock_get_sessions.return_value = ["task-agent-123", "copilot-analysis", "gh-comment-monitor-test"]
         mock_timeout.return_value = {"timeout": False, "reason": "within_timeout"}
 
         # Act
-        with patch('orchestration.cleanup_completed_agents.check_agent_completion') as mock_completion:
+        with patch("orchestration.cleanup_completed_agents.check_agent_completion") as mock_completion:
             mock_completion.return_value = {"completed": False, "reason": "still_active"}
             result = cleanup_completed_agents(dry_run=True)
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)